src/Raven.Server/Documents/Queries/Results/QueryResultRetrieverBase.cs

```diff
@@ -123,6 +123,8 @@ protected void FinishDocumentSetup(Document doc, Lucene.Net.Search.ScoreDoc scor
 
         public abstract bool TryGetKey(ref RetrieverInput retrieverInput, out string key);
 
+        public abstract bool CoraxTryGetKey(IndexSearcher searcher, long id, out UnmanagedSpan key);
```

Comment:
I think that in other places we have convention like `MethodName{SearchEngineType}()` e.g. `UpdateIndexEntriesLucene/UpdateIndexEntriesCorax`. Let's try to follow that so we'll have here `TryGetKeyLucene/TryGetKeyCorax`

Reply:
Done